OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/HelpActivity.java

```diff
@@ -99,4 +104,14 @@ public void onClick(View v) {
     protected void initLayout() {
         setContentView(R.layout.help_activity);
     }
+
+    @IntDef({TAB_START, TAB_FAQ, TAB_CONFIRM, TAB_CHANGELOG, TAB_ABOUT})
+    @Retention(RetentionPolicy.SOURCE)
+    private @interface startHelpActivity {}
```

Comment:
I want to make sure that param code in the method should be one of {TAB_START, TAB_FAQ, TAB_CONFIRM, TAB_CHANGELOG, TAB_ABOUT}(Just for check), I don't know whether this is acceptable.